indico/htdocs/js/indico/modules/registration/registration.js

```diff
@@ -76,8 +76,15 @@
         });
     }
 
+    function getSelectedRows() {
+        var ids = $('.registrations input:checkbox:checked').map(function() {
+            return $(this).val();
+        }).get();
+        return {registration_ids: ids};
```

Comment:
`return ids;` - building the dict should be done when doing the ajax call. it's pretty confusing here, especially with this function name